packages/block-library/src/navigation-link/fallback-variations.js

```diff
@@ -49,6 +50,13 @@ const fallbackVariations = [
 		description: __( 'A link to a tag.' ),
 		attributes: { type: 'tag' },
 	},
+	{
+		name: 'format',
+		icon: formatIcon,
+		title: __( 'Post Format Link' ),
+		description: __( 'A link to a post format.' ),
+		attributes: { type: 'format' },
+	},
```

Comment:
I can't seem to make this work without the fallback 🤔

Reply:
Ah, I see what's happening. First, I need to be on trunk for core. Now after that I have two Tag Link variations. 😕 

Wait, so it looks like Post Formats have the same `item_link` and `item_link_description` as tags 🤯

Someone copied and pasted it years ago and nobody ever noticed?

Reply:
Hmm, ok, so I think there are some defaults in core for tags or categories:
https://github.com/WordPress/wordpress-develop/blob/0df28171ed9c3c31ebc93add82b64df8f690f87c/src/wp-includes/taxonomy.php#L597-L629

Seems unusual.

Reply:
`disablePostFormats` is an editor setting from the `wordpress/editor` package, so this is tricky to use. The navigation editor doesn't use the editor package, and those settings aren't really relevant to it.

Tempted to either remove it, or keep it as is. The impact is quite low either way.

I think I'll remove it for now.